src/modules/secrets/secrets.dal.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { CreateSecretsDto } from './dtos/create-Secrets.dto';
+import { UpdateSecretsDto } from './dtos/update-Secrets.dto';
+
+@Injectable()
+export class SecretsDAL {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async createSecret(createSecretDto: CreateSecretsDto, accountId: number) {
+
+   
+    return await this.prisma.secret.create({
```

Comment:
Again, please format the code and add line breaks to differentiate the operations visually